attachments.md

```diff
@@ -5,21 +5,21 @@
 
 ## Introduction
 
-Matroska supports storage of related files and data in the Attachments Top-Level Element. Attachments can be used to store related cover art, font files, transcripts, reports, error recovery files, picture or text-based annotations, copies of specifications, or other ancilliary files related to the Segment.
+Matroska supports storage of related files and data in the `Attachments Element` (a `Top-Level Element`). Attachments can be used to store related cover art, font files, transcripts, reports, error recovery files, picture or text-based annotations, copies of specifications, or other ancillary files related to the `Segment`.
 
 `Matroska Readers` MUST NOT execute files stored as Attachments.
 
 ## Cover Art
 
-This section defines a set of guidelines for the storage of cover art in Matroska files. A `Matroska Reader` MAY use embedded cover art to display a representation still-image depiction of the multimedia contents of the Matroska file.
+This section defines a set of guidelines for the storage of cover art in Matroska files. A `Matroska Reader` MAY use embedded cover art to display a representational still-image depiction of the multimedia contents of the Matroska file.
 
-Cover art SHOULD only use the JPEG and PNG picture formats.
+Cover art SHOULD only use JPEG and PNG formats.
```

Comment:
Hmm, this sounds a bit… strange. Covert art (images) can not use a format (in the sense that "use" is an active verb). An encoder can use a specific format. Or a picture can be encoded in a format. Maybe something like "Only JPEG and PNG image formats SHOULD be used for cover art pictures." OK, doesn't sound that much better; hopefully I can still get my point across.